launcher/container_runner.go

```diff
@@ -207,6 +218,43 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 	}, nil
 }
 
+// getGRPCClient returns a gRPC verifier.Client pointing to the given address.
+// It also returns a grpc.ClientConn for closing out the connection.
+func getGRPCClient(asAddr string, logger *log.Logger) (verifier.Client, *grpc.ClientConn, error) {
+	opt := grpc.WithTransportCredentials(insecure.NewCredentials())
+	conn, err := grpc.Dial(asAddr, opt)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to open connection to attestation service: %v", err)
+	}
+	return grpcclient.NewClient(conn, logger), conn, nil
+}
+
+// getRESTClient returns a REST verifier.Client that points to the given address.
+// It defaults to the Attestation Verifier instance at
+// https://confidentialcomputing.googleapis.com.
+func getRESTClient(ctx context.Context, asAddr string, spec spec.LauncherSpec) (verifier.Client, error) {
+	httpClient, err := google.DefaultClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create HTTP client: %v", err)
+	}
+
+	var opts []option.ClientOption
+	opts = append(opts, option.WithHTTPClient(httpClient))
+	if asAddr != "" {
+		opts = append(opts, option.WithEndpoint(asAddr))
+	}
+
+	restClient, err := rest.NewClient(context.Background(),
+		spec.ProjectID,
+		spec.Region,
```

Comment:
For now, should we just pass `us-central1` as the region? If we do it's fine to keep the `Region` stuff in the launcher spec.

Reply:
oh sorry you're saying still keep Region in launcher spec because we'll eventually need it. Brainfart on my end.

Reply:
Right now the REST service is only running in us-central1. Ideally, if we were running this in every region, we would do what your code does, call the attestation verifier endpoint corresponding the Region in which the VM is running.

However, right now, we can only call the service in us-central1. To support the ability to run VM in other regions, we can just have those VMs make a cross-region API call to `us-central1`.

Reply:
Basically, for now  we just do:
```go
const defaultRegion = "us-central1"
restClient, err := rest.NewClient(ctx, spec.ProjectID, defaultRegion, opts...)
```
but we keep the code that fills out `spec.Region`.

Reply:
Done. Thanks!